aes/aesni/src/aes128/expand.rs

```diff
@@ -26,10 +26,14 @@ macro_rules! expand_round {
 
 #[inline(always)]
 pub(super) fn expand(key: &[u8; 16]) -> ([__m128i; 11], [__m128i; 11]) {
+    // TODO: eliminate usage of `MaybeUninit` and/or verify soundness
```

Comment:
We probably can replace it with `mem::zeroed()`, since compiler can properly [optimize it](https://rust.godbolt.org/z/inEYXR) (assembly is equivalent to the code using `MaybeUninit`).

Reply:
Nice, looks like that does the trick. Will open a follow-up PR.